packages/ui/src/app/pages/WorkingGroups/WorkingGroup/WorkingGroup.tsx

```diff
@@ -53,13 +54,14 @@ export function WorkingGroup() {
       )
     }
   }
+  const replaceName = () => nameMapping(group?.name ?? name)
```

Comment:
If `replaceName` isn't memoized by useMemo/useCallback, then I think u can just use `nameMapping` in places you need it. It will prevent redeclaring `replaceName` on each render.